pyingest/config/html5.txt

```diff
@@ -0,0 +1,1544 @@
+¡	&iexcl;	&#x000A1;	&#161;	INVERTED EXCLAMATION MARK
```

Comment:
Yes, this is all correct.  The issue with unicode characters appearing in column two is due to some diacritics needing to be called separately from their letter (e.g. n&#773; for an n-overbar).  For now I don't think they impact things but I'll think about how to better define them -- it may be better to define them all for each latin character individually.  They're rare but they happen.